unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Addressables/AddressableResourceProvider.cs

```diff
@@ -45,6 +47,20 @@ public async UniTask<T> GetAddressable<T>(string key, CancellationToken cancella
             return request.Result;
         }
 
+        public async UniTask<T> Instantiate<T>(string address, string name = "", CancellationToken cancellationToken = default)
+        {
+            //This function does nothing if initialization has already occurred
+            await Addressables.InitializeAsync().WithCancellation(cancellationToken);
+
+            AsyncOperationHandle<GameObject> request = Addressables.InstantiateAsync(address);
+            await request.WithCancellation(cancellationToken);
+
+            if(!string.IsNullOrEmpty(name))
+                request.Result.name = name;
```

Comment:
Is it needed, maybe leave it for Editor only?

Reply:
from one hand, it could be useful for logs and exceptions. From another, we can save a bit performance, so I will add Editor check